src/app/contextBroker/CMakeLists.txt

```diff
@@ -88,6 +88,9 @@ ELSEIF(${DISTRO} STREQUAL "openSUSE_13.1")
 ELSEIF(${DISTRO} MATCHES "CentOS_6.*")
   MESSAGE("contextBroker: CentOS DISTRO: '${DISTRO}'")
   TARGET_LINK_LIBRARIES(contextBroker ${STATIC_LIBS} -lmicrohttpd -lmongoclient -lboost_thread-mt -lboost_filesystem-mt -lboost_system-mt -lboost_regex-mt)
+ELSEIF(${DISTRO} MATCHES "CentOS_release")
```

Comment:
Looking to a CentOS 6.6 system, the content of the /etc/redhat-release file is as follows:

```
CentOS release 6.6 (Final)
```

So I understand that, at least in some systems, CentOS 7.x has introduced the `Linux` token.